WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -213,6 +213,12 @@
         sign_in organizer
       end
 
+      it 'cannot confirm competition' do
+        patch :update, id: competition, competition: { name: competition.name }, commit: "Confirm"
```

Comment:
Oh, neat! I did not know about `formaction`. I'm inclined to leave this as is, however.